components/extension-mgt/org.wso2.carbon.identity.extension.mgt/src/test/resources/extensions/applications/template-2/info.json

```diff
@@ -0,0 +1,14 @@
+{
+	"id": "template-1",
```

Comment:
Shall we rename these as template-2 and so on?

Reply:
https://github.com/wso2/carbon-identity-framework/pull/5813/commits/6657f91eb759f0e633afbdc4614bddcdfcb66fe7